### PR TITLE
isCgoImport accounts for back-quoted import string

### PR DESCRIPTION
**Pull request by nishanths**

### CODE REVIEW

File: format/format.go [hunk 1]
```diff
@@ -824,13 +824,21 @@ func identEqual(expr ast.Expr, name string) bool {
...
 // Note that parentheses do not affect the result.
 func isCgoImport(decl *ast.GenDecl) bool {
 	if decl.Tok != token.IMPORT || len(decl.Specs) != 1 {
 		return false
 	}
 	spec := decl.Specs[0].(*ast.ImportSpec)
-	return spec.Path.Value == `"C"`
+	v, err := strconv.Unquote(spec.Path.Value)
+	if err != nil {
+		return false
```
Comment (mvdan) [inline format/format.go:839]: this should be `panic(err) // should never error`

**Review by mvdan — COMMENTED** [review]
The fix itself looks fine, but please add a small test case to testdata/scripts/cgo.txt instead, which will be a much smaller change. Testing this via a CLI integration test is also better than testing it with a very narrow unit test :)

File: format/format.go (hunk 1 above)
Comment (nishanths) [reply format/format.go]: Done.

**Comment by nishanths** [comment]
> please add a small test case to testdata/scripts/cgo.txt instead

Done. Please take a look.

**Review by mvdan — APPROVED** [review]
Thank you!

**Pull request merged**